README.md

```diff
@@ -208,6 +208,7 @@ expressions, in which you can use the literal's keys as JS variables.
   comma as given as a parameter.
 - `exp`: outputs a number in scientific notation, using the number of digits
   after the comma as given as a parameter.
+- `boolean`: escapes boolean values (such as True/False).
```

Comment:
Could you not capitalize `True/False` please? It may be misleading, as it is not capitalized in the output (nor is it capitalized as boolean literals in JS).